Frontend/src/Components/Shared/misc-constants.js

```diff
@@ -1,4 +1,15 @@
 // Time conversion factors
 export const MILLIS_PER_SEC = 1000;
 export const MILLIS_PER_MIN = 60000;
-export const MILLIS_PER_HR = 3600000;
\ No newline at end of file
+export const MILLIS_PER_HR = 3600000;
+
+export const ROUTES = {
+    UPDATE_GRAPHS_METADATA: "/needed-graphs-metadata",
+    GET_SINGLE_VALUES: "/single-values",
+    GET_GRAPH_DATA: "/graph-data",
+    GET_SESSION_LIST: "/sessions-list",
+    CREATE_RECORDING_SESSION: "/create-recording-session",
+    SET_RECORDING_SESSION: "/current-recording-session",
+    SET_RECORD: "/record-data",
+    PROCESS_RECORDED_DATA: "/process-recorded-data"
+}

```

Comment:
I like the use of these ROUTES objects, they seem like an improvement over hardcoding the strings. Would it make sense to share this constant between the backend and the frontend so they don't have to be updated on both sides?

Reply:
I like the thought. Unfortunately, this isn't something we can do since they live in the `src` directories of different packages within the project.

Reply:
Makes sense, thanks!